<?xml version="1.0"?>
<launch>
    <node pkg="cartbot" type="cluster_node" name="cluster_node" output="screen" />

    <node pkg="cartbot" type="tracking_node" name="tracking_node" output="screen" />

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find cartbot)/rviz/cluster.rviz" output = "screen"/>
</launch>